oshi-demo/src/site/site.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    MIT License
+
+    Copyright (c) 2010 - 2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```

Comment:
```suggestion
    Copyright (c) 2010 - 2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```

Reply:
I actually decided some time back that xml files shouldn't be copyrighted.  This is in the [license-plugin excludes list](https://github.com/oshi/oshi/blob/b25a115eee3d79461a5c6c1bcfa52f4e1dd7d19c/pom.xml#L463).

Reply:
no problem, I'll remove, squash and push back up.

Reply:
done, most were already copyrighted - there was just one file that was net new as missing.  Working on the dependabot ability now before this is complete.

Reply:
Yeah, I haven't gone through to remove them yet.  The .properties file headers also eventually need to be axed.